build/pkgs/matroid_database/SPKG.rst

```diff
@@ -0,0 +1,16 @@
+matroid_database: Python interface to matroid database
+======================================================
+
+Description
+-----------
+
+Python interface to matroid database
+
+License
+-------
+
```

Comment:
please add license and more detailed description -- for example by fixing the metadata of the package on PyPI and re-running `sage --package create matroid_database --pypi`

Reply:
I did so manually.